application/testing/CMakeLists.txt

```diff
@@ -1632,8 +1632,6 @@ set_tests_properties(f3d::TestNoFileFileNameTemplate PROPERTIES PASS_REGULAR_EXP
 set_tests_properties(f3d::TestNoFileFileNameTemplate PROPERTIES ENVIRONMENT "CTEST_F3D_NO_DATA_FORCE_RENDER=1")
 
 if(NOT WIN32)
-  f3d_test(NAME TestInvalidCache ENV "HOME=" REGEXP "Could not use default cache directory" NO_BASELINE)
```

Comment:
why not keeping the test ? Its a good test, its just the result of the test that changes

Reply:
It's not testing anything now tbh.  
Maybe I should log a warning if we are trying to cache an HDRI but the cache path is not set.

Reply:
Definitely